test/parallel/test-buffer-alloc.js

```diff
@@ -510,11 +510,10 @@ assert.strictEqual(Buffer.from('=bad'.repeat(1e4), 'base64').length, 0);
   }
 }
 
-// Test single hex character throws TypeError
-// - https://github.com/nodejs/node/issues/6770
-assert.throws(() => Buffer.from('A', 'hex'), TypeError);
+// Test single hex character is discarded.
+assert.strictEqual(Buffer.from('A', 'hex').length, 0);
```

Comment:
Can you add a test where the resulting length is >0 ?
Like `Buffer.from('abx', 'hex')`